src/main/groovy/com/dicedmelon/gradle/jacoco/android/JacocoAndroidPlugin.groovy

```diff
@@ -45,6 +45,10 @@ class JacocoAndroidPlugin implements Plugin<ProjectInternal> {
     plugin
   }
 
+  private static boolean hasKotlin(PluginContainer plugins) {
```

Comment:
Is that method really necessary for one call?

Reply:
It's really not. I was just trying to match the style of [these](https://github.com/fixdauto/jacoco-android-gradle-plugin/blob/ddbd538522f1c2436e522797c4fec85ededf7ec0/src/main/groovy/com/dicedmelon/gradle/jacoco/android/JacocoAndroidPlugin.groovy#L101) (also only used once) but I have no problem in-lining it.